lnwallet/wallet.go

```diff
@@ -186,6 +185,13 @@ type addCounterPartySigsMsg struct {
 	// version of the commitment transaction.
 	theirCommitmentSig []byte
 
+	// This channel is used for the wallet to pass the completed channel
+	// back into the Funding Manager at the final stage in the funding
```

Comment:
Actually, the type of `chanOpen` can be changed to just be a `lnwallet.LightningChannel` and the `OpenChannelDetails` struct can be deleted.